crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -47,7 +47,9 @@ def restore_chef
               if proposal?(filename) && type == :databags
                 bc_name, prop = filename.split("-")
                 prop.gsub!(/.json$/, "")
-                Proposal.create(barclamp: bc_name, name: prop, properties: record.raw_data)
+                proposal = Proposal.first_or_initialize(barclamp: bc_name, name: prop)
```

Comment:
Somehow this doesn't seem to work. While this now updates existing proposals successfully it somehow does not create new one if there wasn't one before. Didn't figure out why this happens yet.